_gradle

```diff
@@ -341,6 +348,7 @@ _gradle() {
         '-Dorg.gradle.logging.level=[Set default Gradle log level.]:log level:(quiet warn lifecycle info debug)' \
         '-Dorg.gradle.parallel=[Set true to enable parallel project builds.]:(true false)' \
         '-Dorg.gradle.priority=[Set priority for Gradle worker processes.]:priority:(low normal)' \
+        '-Dorg.gradle.unsafe.configuration-cache=[Set true to enable Gradle Configuration Cache.]:enable configuration cache:(true false)' \
```

Comment:
There's also `org.gradle.unsafe.configuration-cache-problems=[fail|warn]`

Reply:
Ah sure, can add. Generally, should we even put the system properties in here given they're `unsafe` (and temporary, question mark?). Or should we only recommend the cli flags?

Reply:
Properties are intended for use in `gradle.properties` so I guess we should be fine omitting them in the completion.

Reply:
Makes sense, removed them completely. Out of curiosity, why are the properties called `unsafe` while the flags look like regular flags?

Reply:
It conveys the unsafe nature of the experimental feature. And, given we want to make it enabled by default in the future, it'll make the transition easier by also using new properties without the `unsafe` part.